spacy/tokens/span.pyi

```diff
@@ -115,8 +115,8 @@ class Span:
     end: int
     start_char: int
     end_char: int
-    label: int
-    kb_id: int
+    label: Union[int, str]
+    kb_id: Union[int, str]
```

Comment:
I don't think this is correct. `label` is `int`, `label_` is `str`.

Reply:
The user meant the kwargs for `Span.char_span`.

Reply:
`kb_id` can be both though, right?

Reply:
Nevermind, I misrembered from the [docs](https://spacy.io/api/span) (confused init arguments with argument types).

Reply:
Fixed in https://github.com/explosion/spaCy/pull/12009/commits/cb9bf70a24e83155e3da0609e5fa0d93bd285497.